libs/platform/src/lib/components/button/button.component.html

```diff
@@ -0,0 +1,13 @@
+<button
+    fd-button
+    [options]="options"
+    [compact]="compact"
+    [glyph]="glyph"
+    [disabled]="disabled"
+    class="truncate_btn__txt"
+    (click)="onBtnClick($event)"
+    [fdType]="type"
+    [ngStyle]="{ width: width }"
+>
```

Comment:
Put the width to a quetes

[ngStyle]="{'width': width }"

Reply:
formater is setting it with out '' automatically even tho I am adding '' . If I run with  [ngStyle]="{ 'width' : width }" then build is failing.